README.md

```diff
@@ -18,9 +18,18 @@ npm install eslint-teamcity --save-dev
 ```
 
 ## Usage
+* as onrun eslint formatter:
 ```sh
   eslint --format './node_modules/eslint-teamcity/index.js' myfiletolint.js
 ```
+* as post runnig formatter, work with standart eslint json report
+```sh
+  node index.js result.json
+```
+* as nodejs module
```

Comment:
Maybe the description could something like:
"Require eslint-teamcity and run directly in your JavaScript:`